config/ppo/3DBall_randomize.yaml

```diff
@@ -27,16 +27,11 @@ behaviors:
     threaded: true
 
 parameter_randomization:
-  resampling-interval: 5000
   mass:
-    sampler-type: uniform
```

Comment:
I think I prefer the old format better. On the new one, I think you could do
```
mass:
  uniform:
    min_value: 0.5
    max_value: 10
  gaussian:
    mean: 10
    stddev: 1
```
and then it's ambigious which one to use.

Reply:
We could do something similar to the trainer setting:
```
mass:
  sampler-type: uniform
  sampler-parameters:
    min_value: 0.5
    ma_value: 10
```

Reply:
Yeah, that looks reasonable

Reply:
I updated the config/tests to this^